provider/image_test.go

```diff
@@ -280,7 +391,9 @@ func TestMarshalCachedImages(t *testing.T) {
 
 			"cacheFrom": resource.NewObjectProperty(resource.PropertyMap{
 				"images": resource.NewArrayProperty([]resource.PropertyValue{
-					resource.NewNullProperty(), // unknowns are passed as null property values
+					resource.NewComputedProperty(
+						resource.Computed{Element: resource.NewStringProperty("looking-for-my-image")},
+					),
```

Comment:
You can do this, but you don't need to. 
```suggestion
					resource.MakeComputed(resource.NewStringProperty("looking-for-my-image")),
```

This applies in many test cases.